src/components/suggest_item/suggest_item.test.js

```diff
@@ -4,9 +4,14 @@ import { requiredProps } from '../../test/required_props';
 
 import { EuiSuggestItem } from './suggest_item';
 
+const TYPE = {
```

Comment:
Because we made `type` required, we need to add it to the test so it doesn't fail.